apps/hash-graph/libs/api/src/rpc/account.rs

```diff
@@ -0,0 +1,704 @@
+use alloc::{borrow::Cow, sync::Arc};
+use core::error::{self, Error};
+
+use authorization::{
+    AuthorizationApi as _, AuthorizationApiPool,
+    backend::ModifyRelationshipOperation,
+    schema::{
+        AccountGroupMemberSubject, AccountGroupPermission, AccountGroupRelationAndSubject,
+        WebOwnerSubject,
+    },
+    zanzibar::Consistency,
+};
+use error_stack::{Report, ResultExt as _};
+use graph::store::StorePool;
+use graph_types::{
+    account::{AccountGroupId, AccountId},
+    owned_by_id::OwnedById,
+};
+use harpc_client::{connection::Connection, utils::invoke_call_discrete};
+use harpc_codec::{decode::ReportDecoder, encode::Encoder};
+use harpc_server::{
+    error::{DelegationError, Forbidden},
+    session::Session,
+    utils::{delegate_call_discrete, parse_procedure_id},
+};
+use harpc_service::{delegate::ServiceDelegate, role::Role};
+use harpc_tower::{body::Body, either::Either, request::Request, response::Response};
+use harpc_types::{error_code::ErrorCode, response_kind::ResponseKind};
+use hash_graph_store::account::{
+    AccountStore as _, InsertAccountGroupIdParams, InsertAccountIdParams,
+};
+use temporal_client::TemporalClient;
+
+use super::{role, session::User};
+use crate::rest::PermissionResponse;
```

Comment:
We should probably move out `PermissionResponse` out of `rest` (or redefine it).

Reply:
see: [03d7fb0](https://github.com/hashintel/hash/pull/5577/commits/03d7fb095f6a9ca6c239585c8abc162376a7c5a2)